src/Fixer/ClassNotation/FinalInternalClassFixer.php

```diff
@@ -143,9 +144,13 @@ protected function createConfigurationDefinition()
             (new FixerOptionBuilder('annotation-black-list', 'Class level annotations tags that must be omitted to fix the class, even if all of the white list ones are used as well. (case insensitive)'))
                 ->setAllowedTypes(['array'])
                 ->setAllowedValues($annotationsAsserts)
-                ->setDefault(['@final', '@Entity', '@ORM'])
+                ->setDefault(['@final', '@Entity', '@ORM\Entity'])
```

Comment:
This may or may not be considered a BC break, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3927/files#r268091523
Note: it is not strictly necessary for this PR, I can also explicit `annotation-black-list` in `FinalClassFixer` with the annotations I need, and revert this line, but to me it's worth discussing it

Reply:
tbh i would rather consider this as a bugfix (cc @julienfalque ), and each bugfix is kind-of bc break